Code/Legacy/CryCommon/LoadScreenComponent.cpp

```diff
@@ -353,6 +354,10 @@ void LoadScreenComponent::Stop()
             if (m_loadScreenState == LoadScreenState::Showing)
             {
                 EBUS_EVENT(LoadScreenBus, UpdateAndRender);
+#if defined(CARBONATED)
+                // deliver the log messages
+                AZ::LogNotification::LogNotificatorBus::Broadcast(&AZ::LogNotification::LogNotificatorBus::Events::Update);
+#endif
```

Comment:
Do me a favor and rename "LogNotificatorBus" to "LogNotificationBus" to be more in kind with the rest of the engine.  I see you added the bus back in January, so we should take a moment to bring it closer to O3DE's naming

Reply:
Done after turning off the hooks.